book/src/defmt-versions.md

```diff
@@ -0,0 +1,50 @@
+# Defmt Versions
+
+Any given version of the `defmt` crate will implement one specific *Defmt
+Version* - also known as the *wire format version*. Because a compilation can
+only use one version of the `defmt` crate, each compilation will use a
+consistent *Defmt Version*.
```

Comment:
```suggestion
Any given version of the `defmt` crate will implement one specific *Defmt
Version* - also known as the *wire format version*. Because one compilation can
only use one version of the `defmt` crate, all crates in your dependency tree
must use a consistent *Defmt Version*.
```

Reply:
I think I meant it how I wrote it.

Reply:
Okay. To me that reads a bit confusing but maybe I am just not used to the terms.